pyprep/ransac.py

```diff
@@ -396,3 +411,126 @@ def _get_ransac_pred(
     interpol_mat = _make_interpolation_matrix(reconstr_pos, chn_pos)
     ransac_pred = np.matmul(interpol_mat, data[reconstr_picks, :])
     return ransac_pred
+
+
+def _ransac_correlations_alt(good_data, interpolation_mats, win_size, win_count):
```

Comment:
Can we give this a better name? Potentially even renaming the old "_ransac_correlations", so that we better see what each function does.